pkg/operation/operation.go

```diff
@@ -252,7 +252,7 @@ func (o *Operation) ComputeDownloaderCloudConfig(workerName string) (*chartrende
 		"secretName": o.Shoot.ComputeCloudConfigSecretName(workerName),
 	}
 
-	values, err := o.InjectImages(config, o.K8sShootClient.Version(), map[string]string{"ruby": "ruby"})
+	values, err := o.InjectImages(config, o.TargetShootVersion(), o.TargetShootVersion(), common.RubyImageName)
```

Comment:
Shouldn't we inject the hyperkube image into the config here?

Reply:
I think not: The `shoot-cloud-config-downloader` determines the images it has to preload from the worker secret, otherwise the downloader script would have to change everytime there is a new image.

Reply:
You're right, thanks.